### PR TITLE
site-wide bug fixes 

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: theme/sass/components/_footer.scss [hunk 1]
```diff
@@ -65,52 +65,27 @@
...
                 padding: 0;
                 list-style-type: none;
+                justify-content: center;
 
-                .twitter {
-                    margin-right: 3px;
+                .fa-facebook-f,
+                .fa-twitter,
+                .fa-instagram {
+                    color: white;
```
Comment (ElXyfer) [inline theme/sass/components/_footer.scss]: Should this not be using a variable?

File: theme/sass/components/_footer.scss [hunk 2]
```diff
@@ -65,52 +65,27 @@
...
                 flex-direction: row;
                 padding: 0;
                 list-style-type: none;
+                justify-content: center;
 
-                .twitter {
-                    margin-right: 3px;
+                .fa-facebook-f,
+                .fa-twitter,
+                .fa-instagram {
```
Comment (ElXyfer) [inline theme/sass/components/_footer.scss]: Add a hover state on these so its obvious they're clickable.

File: theme/sass/components/_footer.scss (hunk 1 above)
Comment (katlazarova) [reply theme/sass/components/_footer.scss]: oh yes, so sorry I forgot about this one.

File: theme/sass/components/_footer.scss (hunk 2 above)
Comment (katlazarova) [reply theme/sass/components/_footer.scss]: done

File: theme/sass/components/_footer.scss [hunk 3]
```diff
@@ -65,52 +65,31 @@
...
+                    color: $accent-color-one;
+                    font-size: $font-large;
+                    padding: 10px;
+
+                    &:hover {
+                        cursor: pointer;
```
Comment (ElXyfer) [inline theme/sass/components/_footer.scss]: Maybe change the colour too!

Comment (katlazarova) [reply theme/sass/components/_footer.scss]: okay

**Pull request merged**